docs/src/pages/docs/reference/observability/providers/langsmith.mdx

```diff
@@ -22,14 +24,16 @@ Here's how to configure Mastra to use LangSmith:
 
 ```typescript
 import { Mastra } from "@mastra/core";
+import { AISDKExporter } from "langsmith/vercel";
```

Comment:
are you sure it needs to be vercel?

Reply:
it is according to the docs, unfortunate naming